app/Http/Middleware/HasRoleInService.php

```diff
@@ -38,17 +38,32 @@ public function handle($request, Closure $next)
     /**
      * @param Request $request
      */
-    private function findTheServiceInTheRequest(Request $request)
-    {
-        switch (true) {
+    private function findTheServiceInTheRequest(Request $request) {
+        switch (TRUE) {
```

Comment:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#25-keywords-and-truefalsenull